backends/credhub/src/main/kotlin/org/cloudfoundry/credhub/credentials/DefaultCredentialsHandler.kt

```diff
@@ -42,6 +42,7 @@ class DefaultCredentialsHandler(
     private val credentialGenerator: UniversalCredentialGenerator,
     @Value("\${security.authorization.acls.enabled}") private val enforcePermissions: Boolean,
     @Value("\${certificates.concatenate_cas:false}") var concatenateCas: Boolean,
+    @Value("\${certificates.use_default_key_usages:false}") var useDefaultKeyUsages: Boolean,
```

Comment:
Is there a clearer property name we can use? `use_default_key_usages` sounds a bit too strong, almost like we are overriding user inputs with the defaults, like: use default ones instead of user inputs. 

It also doesn't specify that it applies to CAs only. 

Maybe something like: `enable_default_ca_key_usages`